data/json/monsters/zed_lab.json

```diff
@@ -1,12 +1,13 @@
-
+[
   {
     "id": "mon_zombie_scientist",
     "type": "MONSTER",
     "name": "zombie scientist",
-    "description": "Apart from the jet black eyes it would be easy to believe this scientist was still alive.  Clad in a tattered lab coat, it looks to have some measure of situational awareness and resourcefulness.",
-    "default_faction": "zombie",
+    "description": "Apart from the jet-black eyes, this zombie looks less decomposed and inhuman than most.  Clad in a tattered lab coat, it seems to have a glimmer of awareness - not a human awareness, but something more sinister.  As you watch it, its movements look almost marionette-like.",
+    "default_faction": [ "zombie", "defense_bot" ],
+    "//": "zombie scientists are recognised as friendly by the lab's defense system",
```

Comment:
Do you think it'd be too overpowered for the Lab Technician start to have this, too, through a profession trait?

Reply:
I don't think lab technicians are meant to be XEDRA personnel. There are many other, more common, types of labs.

Reply:
On the contrary, the description of the profession says something along the lines of:
`Can you help undo the very Cataclysm you helped create?`
It's pretty heavily implied that they were employed in these labs, further backed up by the fact that the profession is a valid choice for lab escapes.

Reply:
I would suggest that in general, that's more of a problem with the lab tech description than anything. It shouldn't be an option to be an evacuee who is also a XEDRA lab employee.

In the lab escape, it would be worth considering having the 'lab personnel' get a faction flag that makes them friendly to the automated defenses. 'defense_bot' is not a valid answer though, and personally I think it would be better to write in a reason the player isn't immune to IFF, because of the number of other problems it causes that zombies don't worry about - like being immune to the bots in a totally different XEDRA facility.